### DISCUSSION TITLE
How to manually run?

### DISCUSSION THREAD

**Discussion by derekcentrico** [containrrr/watchtower#901] [Q&A] [answered]
Apologies if this has been answered before, but my searches of the discussions didn't pull up anything on point.

I've run into an issue whereby watchtower was just showing connection refused on all containers and not updating.  I updated the watchtower container which was rather old.  Instead of waiting for the next scheduled run, I would like to test watchtower by running it manually.  I'm trying to find a command or method for doing that.  

Please let me know if it's possible and if so how to manually run watchtower to test it.

**Comment by piksel** [comment, marked as answer]
Just add `--run-once`:
```
docker run -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --run-once
```

**Reply by zirkuswurstikus** [reply]
I also run it with `--rm`.

`--rm` Runs a Docker container and automatically removes it after it exits, ensuring no leftovers.

`docker run --rm -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --run-once`

**Comment by ewoks** [comment]
[alternative with ENV variable](https://containrrr.dev/watchtower/arguments/#run_once): `WATCHTOWER_RUN_ONCE` set to `true`

**Reply by frostbox42** [reply]
Also, don't be silly like me and run it with the restart policy "unless-stopped", this will make it restart itself endlessly.

**Comment by mixailm** [comment]
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --run-once
This will remove ot after that.
